apis/v1alpha1/httproute_types.go

```diff
@@ -87,9 +87,24 @@ type HTTPRouteRule struct {
 	// Match defines which requests match this path.
 	// +optional
 	Match *HTTPRouteMatch `json:"match" protobuf:"bytes,1,opt,name=match"`
-	// Filter defines what filters are applied to the request.
+
+	// Filters define the filters are applied to requests that match
+	// this rule.
+	// Conformance-levels are defined based on the type of filter:
+	// - ALL core filters MUST be supported by all implementations.
+	// - Implementers are encouraged to support extended filters.
+	// - Vendor-specific custom filters have no API guarantees across implementations.
+	//
+	// The effects of ordering of multiple behaviors are currently undefined.
+	// This can change in future based on feedback during the alpha stage.
+	//
+	// Specifying a core filter multiple times has undefined or custom conformance.
+	//
+	// Support: core
+	//
 	// +optional
-	Filter *HTTPRouteFilter `json:"filter" protobuf:"bytes,2,opt,name=filter"`
+	Filters []HTTPRouteFilter `json:"filters" protobuf:"bytes,2,opt,name=filters"`
```

Comment:
I thought based on the doc (https://docs.google.com/document/d/1SkTb6ECuiQiayvumdsZ3sxDcXD6cYs-DKB_C81Ajvh4/edit), we would call this Actions. Is it because Action already exists for ForwardTo?

Reply:
Lists need to have `rep` instead of `opt` protobuf tag.

Reply:
+1 for renaming filters to actions.

Reply:
As per WG discussion yesterday (office hours 12th August), #265 tracks this.